src/libstd/fs.rs

```diff
@@ -756,11 +756,19 @@ impl AsInner<fs_imp::DirEntry> for DirEntry {
 /// guarantee that the file is immediately deleted (e.g. depending on
 /// platform, other open file descriptors may prevent immediate removal).
 ///
+/// # Platform behavior
```

Comment:
I'll wait for some further discussion before going through and editing all of those.